docs/src/index.md

```diff
@@ -0,0 +1,189 @@
+# Introduction
+
+Tree-sitter is a parser generator tool and an incremental parsing library. It can build a concrete syntax tree for a source file and efficiently update the syntax tree as the source file is edited. Tree-sitter aims to be:
+
+- **General** enough to parse any programming language
+- **Fast** enough to parse on every keystroke in a text editor
+- **Robust** enough to provide useful results even in the presence of syntax errors
+- **Dependency-free** so that the runtime library (which is written in pure [C](https://github.com/tree-sitter/tree-sitter/tree/master/lib)) can be embedded in any application
+
+### Language Bindings
+
+There are currently bindings that allow Tree-sitter to be used from the following languages:
+
+- [C#](https://github.com/tree-sitter/csharp-tree-sitter)
+- [Go](https://github.com/smacker/go-tree-sitter)
+- [Guile](https://github.com/Z572/guile-ts)
+- [Haskell](https://github.com/tree-sitter/haskell-tree-sitter)
+- [Java](https://github.com/serenadeai/java-tree-sitter)
+- [Java](https://github.com/bonede/tree-sitter-ng)
+- [Java (Android)](https://github.com/AndroidIDEOfficial/android-tree-sitter)
+- [JavaScript (Node.js)](https://github.com/tree-sitter/node-tree-sitter)
+- [JavaScript (Wasm)](https://github.com/tree-sitter/tree-sitter/tree/master/lib/binding_web)
+- [Kotlin](https://github.com/oxisto/kotlintree)
+- [Lua](https://github.com/euclidianAce/ltreesitter)
+- [OCaml](https://github.com/returntocorp/ocaml-tree-sitter-core)
+- [Odin](https://github.com/laytan/odin-tree-sitter)
+- [Perl](https://metacpan.org/pod/Text::Treesitter)
+- [Python](https://github.com/tree-sitter/py-tree-sitter)
+- [Ruby](https://github.com/Faveod/ruby-tree-sitter)
+- [Ruby](https://github.com/calicoday/ruby-tree-sitter-ffi)
+- [Rust](https://github.com/tree-sitter/tree-sitter/tree/master/lib/binding_rust)
+- [Swift](https://github.com/ChimeHQ/SwiftTreeSitter)
+
+### Parsers
```

Comment:
Can this list be removed and replaced with a link to [this](https://github.com/sogaiu/ts-questions/blob/master/ts-grammar-repositories.txt) (or copied to the wiki)?